common/lib/capa/capa/templates/annotationinput.html

```diff
@@ -59,6 +60,5 @@
 </form>
 
 % if msg:
-<span class="message">${msg|n}</span>
+<span class="message">${HTML(msg)}</span>
```

Comment:
@robrap Neither HTML nor Text is being used where this msg is being constructed. Thanks.